Editor/EditorCore/MenuToggle.cs

```diff
@@ -0,0 +1,198 @@
+﻿
+#if UNITY_2019_1_OR_NEWER
+#define SHORTCUT_MANAGER
+#endif
+
+using System;
+using System.Diagnostics;
+using UnityEngine;
+using UnityEngine.ProBuilder;
+using Debug = UnityEngine.Debug;
+
+namespace UnityEditor.ProBuilder
+{
+    /// <summary>
+    /// Base class from which any action that is represented in the ProBuilder toolbar inherits.
+    /// </summary>
+    public abstract class MenuToggle: MenuAction
+    {
+        public enum MenuToggleState
+        {
+            Inactive,
+            Active,
+        };
+
+        MenuToggleState m_CurrentState;
+
+        protected delegate void StartEndCallBack();
```

Comment:
Use Action<> for delegates https://ono.unity3d.com/unity-extra/unity-meta/raw/@/ReferenceSource/CSharp/Assets/CSharpReference.cs

Reply:
I changed it ! :)